public/main.js

```diff
@@ -13,20 +15,29 @@ form.onsubmit = function(event) {
     event.preventDefault();
 };
 
+tabs.onclick = function () {
+    reloadTodoList();
+};
+
 function createTodo(title, callback) {
-    var createRequest = new XMLHttpRequest();
-    createRequest.open("POST", "/api/todo");
-    createRequest.setRequestHeader("Content-type", "application/json");
-    createRequest.send(JSON.stringify({
-        title: title
-    }));
-    createRequest.onload = function() {
-        if (this.status === 201) {
-            callback();
-        } else {
-            error.textContent = "Failed to create item. Server returned " + this.status + " - " + this.responseText;
-        }
-    };
+    fetch( "/api/todo", {
```

Comment:
i was meant to use the other function i created anyway (createReq), thanks for pointing that out, will fix :)